lib/internal/errors.js

```diff
@@ -118,6 +118,7 @@ E('ERR_HTTP_HEADERS_SENT',
 E('ERR_HTTP_INVALID_STATUS_CODE', 'Invalid status code: %s');
 E('ERR_HTTP_TRAILER_INVALID',
   'Trailers are invalid with this transfer encoding');
+E('ERR_ICU_NOT_SUPPORTED', '%s is not supported on Node.js built without ICU');
```

Comment:
The reasoning was "test the message once then assume it's ok everywhere else" but, this will probably get reworked anyway after we finish migrating all the Error, so this is non-blocking.